docs/npm.md

```diff
@@ -0,0 +1,395 @@
+# npm
+
+<https://docs.npmjs.com/>
+
+* Overview [in the README][readme-npm]
+* [Specifying packages to process](#specifying-packages-to-process)
+* [npm dependencies](#npm-dependencies)
+* [Required files](#required-files)
+* [Using fetched dependencies](#using-fetched-dependencies)
+
+## Specifying packages to process
+
+A package is a file or directory that is described by a package.json file.
+
+* The required files for npm are package.json and one of package-lock.json or npm-shrinkwrap.json. See [Required files](#required-files) and npm documentation:
+  * See [package.json](https://docs.npmjs.com/cli/v9/configuring-npm/package-json)
+  * See [package-lock.json](https://docs.npmjs.com/cli/v9/configuring-npm/package-lock-json)
+
+The package-lock.json version must be **higher than v1**! package-lock.json v1 is not supported in Cachi2.
+The last npm version associated with package-lock.json v1 was in nodejs v14 (which was shipped with npm version 6), which is EOL since 30 Apr 2023.
+See [nodejs endoflife.date](https://endoflife.date/nodejs).
+
+Cachi2 fetch-deps shell command:
+
+```shell
+cachi2 fetch-deps \
+  --source ./my-repo \
+  --output ./cachi2-output \
+  '<JSON input>'
+```
+
+JSON input:
+```jsonc
+{
+  "type": "npm",
+  // path to the package (relative to the --source directory)
+  // defaults to "."
+  "path": ".",
+  // specify path to required files (relative to the package path)
+  // where "npm-shrinkwrap.json" or "package-lock.json" is present.
+  // When both are present, npm-shrinkwrap.json takes precedence.
+}
+```
+
+## npm dependencies
+
+The "npm package" formats that Cachi2 can process are the following:
```

Comment:
And if the "Required files" section looks a bit empty with the above change, this would be a good subsection that explains the possible dependency types in package.json / package-lock.json

Reply:
OK moved to `Required files` subsesction